index.js

```diff
@@ -58,7 +58,15 @@ class MobileFriendly {
       console.log(`Checking site ${this.url} ...`);
 
       request.post(options, (err, data) => {
-        const body = JSON.parse(data.body);
+        let body;
```

Comment:
If API returns a HTTP 502 error, does data consist of error code or smth? If no, I more than happy to merge it.